src/client/contextmenu.js

```diff
@@ -68,11 +68,14 @@ export default class ContextMenu {
       }, "CMD+K"],
       ["Browser",     (evt) => {
         this.openComponentInWindow("lively-container", evt).then(comp => {
-          var container = _.last(document.querySelectorAll("lively-container"));
+          let containers = Array.from(document.querySelectorAll('lively-container'))
```

Comment:
sounds good... and the new syntax seems also nice (when it will be
available...)

2016-07-20 10:44 GMT+02:00 Jan Graichen <notifications@github.com>:

> In src/client/contextmenu.js
> <https://github.com/LivelyKernel/lively4-core/pull/73#discussion_r71486500>
> :
>
> > @@ -68,11 +68,14 @@ export default class ContextMenu {
> >        }, "CMD+K"],
> >        ["Browser",     (evt) => {
> >          this.openComponentInWindow("lively-container", evt).then(comp => {
> > -          var container = _.last(document.querySelectorAll("lively-container"));
> > +          let containers = Array.from(document.querySelectorAll('lively-container'))
>
> Using the new loader w/ Babel 6 one could even use the new function bind
> syntax making it like functional composition / pipe operator:
>
> files::filter(e => e.name.match(...))::sortBy(e => e.name)::last()
>
> But I did not get it to work with the legacy boot even when replacing and
> upgrading babel to babel-standalone etc. So for now it has to be functional:
>
> last(sortBy(filter(files, e => e.name.match(...)), e => e.name))
>
> Will drop same patches today or next week.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LivelyKernel/lively4-core/pull/73/files/8924ae90e466db8078e83a61991c9ae52226680e#r71486500>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABZfh-ec1cMBW5vxQPsxrMJCL800CFAGks5qXd_VgaJpZM4JPstw>
> .
>